src/algorithms/discretevoronoi.jl

```diff
@@ -0,0 +1,32 @@
+"""
+    DiscreteVoronoi
+
+Assigns a value to each patch using a 1-NN algorithmm with `n` initial clusters
+and `1` neighbors. The default is to use three clusters.
+"""
+struct DiscreteVoronoi <: NeutralLandscapeMaker
+    n::Int64
+    function DiscreteVoronoi(n::Int64)
+        @assert n > 0
+        new(n)
+    end
+end
+
+DiscreteVoronoi() = DiscreteVoronoi(3)
+
+function _landscape!(mat, alg::DiscreteVoronoi)
```

Comment:
Couldn't this whole function just be 
```julia
_landscape!(mat, alg::DiscreteVoronoi) = _landscape!(mat, NearestNeighborElement(alg.n, 1))
```
end of story?

Reply:
Indeed. We probably don't need to reimplement. Just add in documentation that `NearestNeighborElement` with `k` set at 1 is voronoi-like.

Reply:
Basically, I was just looking for a way to rasterize a Voronoi (#30). Now I know it's super easy using `NearestNeighborElement`, we can close the issue 👍

Reply:
Ah no, I think it's great to have a wrapper for this! i just wanted to reuse the function we already had. I think it should work if you edit the PR as suggested :-)

Reply:
Sorry I wasn't clear what I meant by "reimplement"